packages/blade/src/components/Popover/_decisions/decisions.md

```diff
@@ -0,0 +1,275 @@
+      <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+    </Popover>
+  )
+}
+```
+
+#### Working with non-interactive triggers
+
+To make popover apear on clicking a non-interactive elements such as icons, badges, counters etc. We will provide a `PopoverInteractiveWrapper` component which will work as a minimal trigger:
+
+```js
+import { Popover, PopoverInteractiveWrapper } from '@razorpay/blade';
+
+// non-interactive element as trigger
+<Popover content="Amount reversed to customer bank account">
+  <PopoverInteractiveWrapper>
+    <Counter value={100} />
+  </PopoverInteractiveWrapper>
+</Popover>;
+```
+
+> Note: If users don't wrap the non-interactive elements in `PopoverInteractiveWrapper` the popover won't apear
+
+### Custom Triggers
+
+Users will be able to use their own custom components as triggers:
+
+To make custom elements work the components needs to expose it's inner ref, this can be achived by using `React.forwardRef`:
+
+```jsx
+import type { PopoverTriggerProps } from "@razorpay/blade/components";
+
+type MyCustomButtonProps = {
+  children: React.ReactNode
+} & PopoverTriggerProps
+
+const MyCustomButton = React.forwardRef<
+  HTMLDivElement,
+  MyCustomButtonProps
+>(({ children, ...props }, ref) => {
+  return (
+    <button
+      ref={ref}
+      tabIndex={0}
+      // you can either spread the props
+      // or pick the onMouseDown, onMouseUp.. etc events and add them individually
+      {...props}
+    >
+       {children}
+    </button>
+  );
+});
+
+// Custom trigger
+<Popover content="Amount reversed to customer bank account">
+  <MyCustomButton>Click me<MyCustomButton>
+</Popover>
+```
+
+## Future Scope
+
+Apart from normal Popovers while we audited the components used in Razorpay, we also found instances of Guided Popovers (Tour component), which can be used to guide users through a flow. 
+We will be adding support for Guided Popovers in future.
+
+Guided Popovers will have additional features like: 
+
+- Ability to add multiple steps in a flow without juggling with state management
+- Mask & backdrop support
+- Proper guidelines on how to use the component
+- etc
+
+This will be a separate component and will be released in future, since the scope & complexity of Guided Popovers will be much larger than normal Popovers.
+
+## Tooltips vs Popovers vs Guided Popovers
+
+|                         	| Tooltip                                          	| Popover                                                        	| GuidedPopover                                                  	|
+|-------------------------	|--------------------------------------------------	|----------------------------------------------------------------	|----------------------------------------------------------------	|
+| **Use Case**            	| Set context                                      	| Set context<br>Let user take action                            	| Highlight new feature<br>Let users take action                 	|
```

Comment:
add a point for dismiss interaction?

Reply:
Already added at the end. "Dismiss Button"

Reply:
no i mean dismiss button change handler

Reply:
the change handler is a implementation detail / prop, don't think it qualifies as a distinctive difference. 

If there is not Dismiss button, it's implied that there can't be onDismissButtonClick.